application/config/cloudlog.php

```diff
@@ -32,7 +32,7 @@
 | Default is: M
 |
 */
-$config['measurement_base'] = 'M';
+$config['measurement_base'] = 'K';
```

Comment:
I think we should leave the defaults here.

Reply:
> I think we should leave the defaults here.

Done